src/V3PreProc.cpp

```diff
@@ -316,6 +317,14 @@ bool V3PreProcImp::defExists(const string& name) {
     const auto iter = m_defines.find(name);
     return (iter != m_defines.end());
 }
+bool V3PreProcImp::defCmdline(const string& name) {
+    const auto iter = m_defines.find(name);
+    if (iter == m_defines.end()) {
+        fileline()->v3error("Define or directive not defined: `" + name);
+        return "";
+    }
+    return iter->second.cmdline();
```

Comment:
I'd think you just return true or false, rather than error, given where this is used?